4.figures/main_figure_1/scripts/3.main_figure_1.r

```diff
@@ -0,0 +1,62 @@
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(ggplot2))
+suppressPackageStartupMessages(library(grid))
+suppressPackageStartupMessages(library(patchwork))
+suppressPackageStartupMessages(library(RColorBrewer))
+
+figure_dir <- "../figures"
+output_main_figure_1 <- file.path(figure_dir, "main_figure_1_workflow.png")
+
+workflow_path = file.path("../figures/workflow.png")
+workflow_img = png::readPNG(workflow_path)
```

Comment:
Is the `png` module part of the R-library or is it required to be installed? I've noticed the `png` library is not an imported module in your notebook. 

Could it be part of the `r-essentials` package?

Reply:
I am not 100% sure, unfortunately! I do believe it is a part of one of these libraries, if I had a guess. I had asked Mike about this before, and he used it originally in pyroptosis repo so that is where I had found the code.